source/components/home/navigation/navigation.js

```diff
@@ -32,13 +37,16 @@ class Navigation extends Component {
                         <Navbar.Toggle />
                     </Navbar.Header>
                     <Navbar.Collapse>
-                    <Nav pullRight>
+                    <Nav>
                         <NavItem onClick={() => this.scrollTo('About')} eventKey={1} className="Navigation__item--active">About</NavItem>
                         <NavItem onClick={() => this.scrollTo('Speakers')} eventKey={2} >Speakers</NavItem>
                         <NavItem onClick={() => this.scrollTo('Sponsors')} eventKey={3} >Sponsors</NavItem>
                         <NavItem onClick={() => location.href="https://blog.reflectionsprojections.org"} eventKey={4}>Blog</NavItem>
                         <NavItem onClick={() => location.href="/#/faq"} eventKey={5}>FAQ</NavItem>
                     </Nav>
+                    <Nav bsStyle="pills" pullRight>
+                        <Button onClick={() => render(<Registration/>)} bsStyle="success" bsSize="large">Register</Button>
```

Comment:
Modal doesn't pop up when I click the button :/

Reply:
now it does